src/leechkit/__main__.py

```diff
@@ -18,7 +18,8 @@ def main(
     collection: Path,
     query: str = "deck:current",
     tag: str = "maybe-leech",
-    skip_reviews: int = 3,
+    skip_reviews: int = 1,
```

Comment:
Let's not change the default for now

```suggestion
    skip_reviews: int = 3,
```